impl/src/main/java/org/glassfish/json/JsonReaderImpl.java

```diff
@@ -36,23 +36,28 @@
  * @author Jitendra Kotamraju
  */
 class JsonReaderImpl implements JsonReader {
+    public static final String ALLOW_DUPLICATE_KEYS = "org.glassfish.json.JsonReaderImpl.allowDuplicateKeys";
```

Comment:
does it make sense to have `public static` field in a class with `package` level visibility?

Reply:
I have added this configuration property to the JsonReader interface in the api module.